### DISCUSSION TITLE
Support for Windows Forms App, samples

### DISCUSSION THREAD

**Discussion by cdecinkoKnight** [keycloak/keycloak#13871] [Q&A]
Where can I find documentation on using Keycloak with a Windows Forms desktop application? Perhaps some sample code?

**Comment by bonnm** [comment]
Hi,
[https://www.nuget.org/packages/IdentityModel.OidcClient/](https://www.nuget.org/packages/IdentityModel.OidcClient/) is a package I used for convenient standalone client applications. Without third-party libraries, the [https://www.nuget.org/packages?q=Microsoft.IdentityModel](https://www.nuget.org/packages?q=Microsoft.IdentityModel) packages contain all you need.
